src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +12,52 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static final class DriveConstants {
+        // --> CAN IDs (motors)
+        public static final int kLeftDriveCAN1 = 0;
+        public static final int kLeftDriveCAN2 = 0;
+        public static final int kLeftDriveCAN3 = 0;
+
+        public static final int kRightDriveCAN1 = 1;
+        public static final int kRightDriveCAN2 = 1;
+        public static final int kRightDriveCAN3 = 1;
+
+        // Voltage Regulation
+        //public static final double kSlowVoltage = 0; // TODO: Replace placeholder value
+        public static final double kNormalVoltage = 5.25;
+        public static final double kFastVoltage = 10.5;
+
+        // Power Control
+        public static final double kSecondsToReachFullSpeed = 0.25; 
+        public static final double kTurnThreshold = 0.2;
+        
+        // Encoders
+        public static final int[] kLeftDriveEncoderPorts = new int[] {0, 1};
```

Comment:
This is unused since we are using the built-in encoders on the Falcons

Reply:
Yes, this will be removed as long as removing it does not interfere with other functionality.